app/src/main/java/com/codesoom/assignment/enums/HttpStatusCode.java

```diff
@@ -0,0 +1,32 @@
+package com.codesoom.assignment.enums;
+
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+ * https://datatracker.ietf.org/doc/html/rfc7231#section-6.1
+ * */
```

Comment:
MDN 문서도 결국은 RFC를 읽고 쓴 문서죠. 

```suggestion
/**
 * https://datatracker.ietf.org/doc/html/rfc7231#section-6.1
 * */
```